src/test/java/kitchenpos/domain/TableGroupTest.java

```diff
@@ -0,0 +1,47 @@
+package kitchenpos.domain;
+
+import kitchenpos.table.domain.OrderTable;
+import kitchenpos.tablegroup.domain.TableGroup;
+import kitchenpos.tablegroup.exception.IllegalOrderTablesException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class TableGroupTest {
+
+    @DisplayName("테이블이 없는 테이블 그룹을 생성한다")
+    @Test
+    void emptyOrderTableTest() {
+        assertThatThrownBy(() -> new TableGroup(Collections.emptyList())).isInstanceOf(IllegalOrderTablesException.class);
+    }
+
+    @DisplayName("테이블이 한 개 있는 테이블 그룹을 생성한다")
+    @Test
+    void oneOrderTableTest() {
+        OrderTable orderTable1 = new OrderTable(null, 1, true);
+        assertThatThrownBy(() -> new TableGroup(Collections.singletonList(orderTable1))).isInstanceOf(IllegalOrderTablesException.class);
+    }
+
+    @DisplayName("빈 테이블이 아닌 테이블을 테이블 그룹으로 생성한다")
+    @Test
+    void NoEmptyTableTest() {
+        OrderTable orderTable1 = new OrderTable(null, 1, true);
```

Comment:
```suggestion
        OrderTable emptyOrderTable = new OrderTable(null, 1, true);
```

1, 2 보다는 네이밍을 통해 의도를 나타낼 수 있으면 좋을 것 같아요~